google-account-plugin/testSrc/com/google/cloud/tools/intellij/stats/UsageTrackerManagerTest.java

```diff
@@ -34,7 +36,7 @@
  * Tests for {@link UsageTrackerManager}.
  */
 @RunWith(MockitoJUnitRunner.class)
-public class UsageTrackerManagerTest extends TestCase {
+public class UsageTrackerManagerTest {
```

Comment:
@joaoandremartins could you make this change in a separate PR? I don't want to wait to merge this and get rid of the warning.